utils/nctl/sh/scenarios/itst14.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+
+source "$NCTL"/sh/scenarios/common/itst.sh
+
+# Exit if any of the commands fail.
+set -e
+
+#######################################
+# Runs an integration tests that tries to simulate
+# if a validator node can restart within single era 
+# and not equivocate.
+# Arguments:
+#   `timeout=XXX` timeout (in seconds) when syncing.
+#######################################
+function main() {
+    log "------------------------------------------------------------"
+    log "Starting Scenario: itst14"
+    log "------------------------------------------------------------"
+
+    # 0a. Verify all nodes are in sync
+    check_network_sync
+    log "Sleeping 10s"
+    sleep 10
+    check_network_sync
```

Comment:
I would replace these with `nctl-await-n-blocks 5`, (as long as 5 is less than the minimum era height from the chainspec) to verify that network is capable of proposing blocks.

Reply:
didnt even know that one existed, thanks!

Reply:
added a new function to common for this. I see it being reused again.